rs/backend/src/canisters/governance.rs

```diff
@@ -14,3 +17,53 @@ pub async fn claim_or_refresh_neuron_from_account(
     .await
     .map_err(|e| e.1)
 }
+
+#[cfg(not(test))]
+pub use prod::get_metrics;
+
+#[cfg(test)]
+pub use testing::get_metrics;
+
+#[cfg(not(test))]
+mod prod {
+    use super::{candid, GovernanceCachedMetrics, GovernanceError, GOVERNANCE_CANISTER_ID};
+
+    pub async fn get_metrics() -> Result<Result<GovernanceCachedMetrics, GovernanceError>, String> {
+        dfn_core::call(GOVERNANCE_CANISTER_ID, "get_metrics", candid, ())
+            .await
+            .map_err(|e| e.1)
+    }
+}
+
+#[cfg(test)]
+pub mod testing {
+    use super::*;
+    use std::{cell::RefCell, collections::VecDeque};
+
+    thread_local! {
+        pub static RESPONSES:
+        RefCell<VecDeque<Result<Result<GovernanceCachedMetrics, GovernanceError>, String>>> = RefCell::default();
```

Comment:
You might want to break out the element type, like so:

```
type GetMetricsResult = Result<Result<GovernanceCachedMetrics, GovernanceError>, String>;
```

Then, here, you can say

```suggestion
        RefCell<VecDeque<GetMetricsResult>> = RefCell::default();
```

I'm not super in favor of this, because I believe type aliases usually lead to confusion (because of "extra" indirection). But in this case, there is a ton of nesting, so it might be worthwhile. Also, the type alias that I suggested can be used an a couple other places. If you adopt alias, use it throughout.